instance/autobuy.go

```diff
@@ -16,7 +16,7 @@ func (in *Instance) abLaptop(_ discord.Message) {
 	if trigger == nil || trigger.Value != postmemeCmdValue {
 		return
 	}
-	in.sdlr.ResumeWithCommand(&scheduler.Command{
```

Comment:
You said you changed this to `PrioritySchedule` because of a bug. But then instead of implementing a temporary fix like this the underlying bug should be fixed.

Reply:
I'll look into ResumeWithCmd function

Reply:
I found the bug. Basically as it does resume's with a command, what was happening that "pls hunt" command was finished after it got the output and when it tried to "pls buy rifle 1", as pls hunt was already finished, it decided to resume it with the next command which was "pls search" and which would result in a cancellation loop due to "not a valid search option"

Reply:
I changed stuff to "ResumeWithCommandOrPrioritySchedule". Now it works gr8

Reply:
> I found the bug. Basically as it does resume's with a command, what was happening that "pls hunt" command was finished after it got the output and when it tried to "pls buy rifle 1", as pls hunt was already finished, it decided to resume it with the next command which was "pls search" and which would result in a cancellation loop due to "not a valid search option"

Don't exactly understand this. For the hunt command, this part:
```go
trigger := in.sdlr.AwaitResumeTrigger()	
if trigger == nil || trigger.Value != huntCmdValue {
	return
}
```
should ensure that the scheduler is actually awaiting a resume. The rest of the function should not be reached otherwise. This is because if the scheduler was not awaiting a resume, `in.sdlr.AwaitResumeTrigger()` would return `""`, therefore causing `trigger.Value != huntCmdValue` to be `true` and the function to return.

Reply:
i'll give it another test and let you know

Reply:
just checked it there definitely is a bug, I am not skilled enough to identify, It executes some other command and then it tries to buy the rifle. To reproduce, change it to "ResumeWithCommand" and watch the bot get stuck for 5-10 secs when it sees you dont have a rifle then it would do the bug. Please look into this yourself. you understand your code best. The new auto dig PR is also affected by this bug

Reply:
can we accept this as a temporary fix until the bug is found?